internal/pkg/deploy/cloudformation/stack/backend_svc.go

```diff
@@ -87,6 +87,14 @@ func (s *BackendService) Template() (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("convert storage options for service %s: %w", s.name, err)
 	}
+	entrypoint, err := s.manifest.EntryPoint.ToStringSlice()
+	if err != nil {
+		return "", fmt.Errorf("convert entrypoint to string slice: %w", err)
```

Comment:
nit (same for the other errors):
```suggestion
		return "", fmt.Errorf(`convert 'entrypoint' to string slice: %w`, err)
```